O/OpenCL/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "OpenCL"
+version = v"2022.9.23"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/KhronosGroup/OpenCL-Headers.git", "4c50fabe3774bad4bdda9c1ca92c82574109a74a"),
+    GitSource("https://github.com/KhronosGroup/OpenCL-ICD-Loader.git", "3dae4803532b11d74e4dc216ee72570c1a4bff24")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+
+install_license ./OpenCL-Headers/LICENSE
+
+wget https://patch-diff.githubusercontent.com/raw/KhronosGroup/OpenCL-Headers/pull/209.patch
+patch ./OpenCL-Headers/tests/test_headers.c 209.patch
```

Comment:
Either add this URL as a new [`FileSource`](https://docs.binarybuilder.org/stable/reference/#BinaryBuilderBase.FileSource) or copy the patch in a `bundled/patches` subdirectory and add `./bundled` as [`DirectorySource`](https://docs.binarybuilder.org/stable/reference/#BinaryBuilderBase.DirectorySource).  There are hundreds of similar examples in Yggdrasil.